src/main/webapp/app/shared/service/sort.service.ts

```diff
@@ -105,6 +105,11 @@ export class SortService {
             if (!obj) {
                 return obj;
             } else {
+                // Function calls without arguments (e.g. last()) are also supported.
+                if (key.endsWith('()')) {
+                    const functionName = key.substring(0, key.length - 2);
+                    return obj[functionName]?.();
+                }
```

Comment:
- Exists is covered by `?.()`
- As for runtime errors in case `obj[functionName]` is not callable - perhaps this is sensible here (as it indicates that whoever provided the path to the desired attribute/function made a mistake with the types)